src/clj/ataru/hakija/hakija_application_service.clj

```diff
@@ -274,12 +274,14 @@
                                                              (:haku application)
                                                              (util/application-in-processing? (:application-hakukohde-reviews application))
                                                              form-roles)
-                               (some? (:form application)) (hakija-form-service/fetch-form-by-key
+                               (some? (:form application)) (hakija-form-service/fetch-form-by-key-with-flagged-fields
                                                              (->> application
                                                                   :form
                                                                   form-store/fetch-by-id
                                                                   :key)
-                                                             form-roles))
+                                                             form-roles
+                                                             nil
+                                                             (util/application-in-processing? (:application-hakukohde-reviews application))))
```

Comment:
Tota `:in-processing-state-in-jatkuva-haku` rivillä 292 pitää sorvata kans niin, että kälissä voidaan näyttää oikeet tekstit. Sen vois nimetä vaik `:cannot-edit-because-in-processing` tai jotain.